bin/glite-info-update-endpoints

```diff
@@ -211,7 +237,7 @@ if __name__ == "__main__":
             sys.exit(1)
         egi_urls = get_egi_urls(config['certification_status'])
         pickle_file = config['cache_dir'] + '/' + 'EGI.pkl'
-        if len(egi_urls) > 0:
+        if egi_urls != "" and len(egi_urls) > 0:
```

Comment:
Can you explain what runtime condition led you to add this extra conditional ?

Reply:
Well, I did not want to modify original interface of the `get_osg_urls` and `get_egi_urls` functions event thought I did not like it - normally these functions returns dictionary but in case of Exception during HTTP query empty string is returned and in such case I guess we just want to read older data that were successfully read in the past.

Reply:
Thank you for explaining that. That's  a really important consideration... again, I think we should be adding some comments for why functions do what they do and then refactor once test coverage has been added to improve the implementation.